packages/react-img-viewer/README.md

```diff
@@ -0,0 +1,160 @@
+# TSDX React User Guide
```

Comment:
**Chore**
This looks like tsdx generated readme. Need to be updated with react-img-viewer specific readme.